assets/js/logic.js

```diff
@@ -14,25 +14,6 @@
   var videoIdArray = ["X5Cfi7U4eL4"]
   var imageArray = ["https://i.ytimg.com/vi/X5Cfi7U4eL4/hqdefault.jpg",]
 
-  $.ajax({
-    url: "https://www.googleapis.com/youtube/v3/search?key=AIzaSyB213LbFGRmVEeWvM3O3-AVreKCX6uhJXk&q="
-    + artist + "&"+ song + "&part=snippet&type=video&videoDuration=" + duration + "&videoSyndicated=true&maxResults=8",
-    method: "GET"
-    }).then(function(response) {
-    for (i = 0; i < 8; i++){
-        console.log(response.items[i].id.videoId);
-        videoIdArray.push(response.items[i].id.videoId);
-        console.log(response.items[i].snippet.title);
-        titleArray.push(response.items[i].snippset.title);
-        console.log(response.items[i].snippet.thumbnails.high.url);
-        imageArray.push(response.items[i].snippet.thumbnails.high.url);
-        };
-        console.log(titleArray);
-        console.log(videoIdArray);
-        console.log(imageArray);
-        //displayImages();
-    });
-
 $(function() {
```

Comment:
What happened to the rest of the animation code? I can't see it on my phone, is it still there? I only see the onload function and the comment plus two lines of code

Reply:
I see all of it here in GitHub.